### ISSUE TITLE
DOC: What is the maximum chunk size returned from SemanticChunker.split_documents()

### ISSUE THREAD

**Issue by abhipandey09** [langchain-ai/langchain#28250]
### Checklist

- [X] I added a very descriptive title to this issue.
- [X] I included a link to the documentation page I am referring to (if applicable).

### Issue with current documentation:

I need to know on the maximum chunk size that can be return from SemanticChunker.split_documents() for large documents.
Can it be more than 8k token as i need to send the chunk for embedding, chunk with more than 8k token will fail with Azure embedding model.
Need Help!!

### Idea or request for content:

Documentation should cleary explain this

**Comment by keenborder786** [comment]
@abhipandey09, I don't think so there is any upper limit as such. But it depends of number_of_chunks. If `number_of_chunks` is high then each chunk size will be lower and vice versa so you can adjust that parameter.
